backend/fms_core/services/study.py

```diff
@@ -0,0 +1,39 @@
+from django.core.exceptions import ValidationError
+from fms_core.models import Study
+
+def create_study(letter, project, worfklow, start, end, reference_genome=None):
+    study = None
+    errors = {}
+    warnings = {}
+
+    if not letter:
+        errors['letter'] = 'Missing letter.'
+        return study, errors, warnings
+
+    if not project:
+        errors['project'] = 'Missing project.'
+        return study, errors, warnings
+
+    if not worfklow:
+        errors['workflow'] = 'Missing workflow.'
+        return study, errors, warnings
+
+    if not start:
```

Comment:
I was wondering if we could accept default values as input and fill then internally at insertion time so when we want to test a sample against it's study workflow the information is calculated, but without forcing default entries (the start and end of the full workflow)